source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,201 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
+ - ``pas`` and ``pascal`` code blocks are now higlighted.
+ - Language setting in Boards was removed. The main language setting under **Settings -> Display Settings** now covers Boards. Boards previously supported Catalan, Greek, Indonesian, and Occitan, but since these 4 languages were only partially translated (<40%; Boards-only), they have been removed until more areas of Mattermost are translated into these languages.
+ - Removed websocket state effects for the collapse/expand state of categories.
+ - Pre-packaged Calls version 0.16.1.
+ - Pre-packaged Jira plugin version 3.2.5.
+ - Pre-packaged GitHub plugin version 2.1.6.
+ - Pre-packaged Autolink plugin version 1.4.0.
+ - Pre-packaged Welcomebot plugin version 1.3.0.
+ - Pre-packaged NPS plugin version 1.3.2.
+ - Added support to specify different desktop notification sounds per channel.
+ - Calls: Ringing sounds can be enabled/disabled and selected in the **Desktop Notifications** preferences panel.
+
+#### Administration
+ - Added a new ``ConfigurationWillBeSaved`` plugin hook which is invoked before the configuration object is committed to the backing store.
+ - Admins can now specify index names to ignore while purging indexes from Elasticsearch with the ``ElasticsearchSettings.IgnoredPurgeIndexes`` setting.
+ - Added an option to use the German HPNS notification proxy.
+ - New flags were added to the [database migrate command](https://docs.mattermost.com/manage/command-line-tools.html#mattermost-db-migrate) as following:
+
+    - ``auto-recover``: If the migration plan receives an error during migrations, this command will try to rollback migrations already applied within the plan. This option is not recommended to be added without reviewing migration plan. You can review the plan by combining ``--save-plan`` and ``--dry-run`` flags.
+    - ``save-plan``: The plan for the migration will be saved into the file store so that it can be used for reviewing the plan or to be used for downgrading.
+    - ``dry-run``: Does not apply the migrations, but it validates how the migration would run with the given conditions.
+
+ - A new [database subcommand](https://docs.mattermost.com/manage/command-line-tools.html#mattermost-db-downgrade) "downgrade" was added to be able to rollback database migrations. The command either requires an update plan to rollback, or comma separated version numbers.
+ - Removed ``/api/v4/users/stats`` network request from ``InviteMembersButton``.
+ - Self-hosted admins can now define a separate shipping address during in-product license purchase.
+ - Added updates to the trial request forms to allow for a more tailored trial experience.
+ - First admins will now have an onboarding experience that includes first team creation based on company name and invite members link steps. 
+ - Adds the ability to expand seats in-product for self-hosted servers.
+ - Added a new section in the **System Console > Products** for Boards.
```

Comment:
Shouldn't this setting be in the plugin settings @sbishel rather than in a new system console section? If you agree perhaps we should add a ticket for v9.0